rust/src/python/libnmstate/clib_wrapper.py

```diff
@@ -200,6 +200,31 @@ def gen_conf(state):
     # pylint: enable=no-member
 
 
+def gen_diff(desired_state, current_state):
```

Comment:
IMHO it is more typical to have the old (current state) as the first and the new (desired state) as the second argument, similar to the "diff" shell command.

Reply:
Sure!